demo/demo-tekton.sh

```diff
@@ -258,8 +292,10 @@ oc process -f "$tekton_setup_dir/05-build-and-deploy.yaml" -p PROJECT_URI="${GIT
            -p NAMESPACE="${app_namespace}" -p APPLICATION_NAME="${app_name}" \
            -n default > /tmp/05-build-and-deploy.yaml.out 2>/tmp/05-build-and-deploy.yaml.err
 oc apply -n "${app_namespace}" -f /tmp/05-build-and-deploy.yaml.out
+# should delete files from /tmp/?
 
 route=$(oc get -n "${app_namespace}" "route/${app_name}" --output=go-template='http://{{.spec.host}}')
```

Comment:
No, this is the route to the application that is being built.

Reply:
But shouldn't we print Grafana route too? Since its a demo, a believe the most important part is to show the dashboards to the user